package.json

```diff
@@ -5,6 +5,7 @@
   "main": "./dist/store.js",
   "scripts": {
     "test": "npm run build_test && jasmine",
+    "clean-ng2": "rimraf node_modules/angular2/manual_typings node_modules/angular2/typings",
```

Comment:
I'm about to push with `rimraf` included as dep and a cleanup to the `typings.json` to reflect latest changes in typings package names and commits.